cmd/mta.go

```diff
@@ -209,6 +245,22 @@ var updateBuildParametersCmd = &cobra.Command{
 	SilenceErrors: true,
 }
 
+// updateGlobalParametersCmd update global parameters in mta
+var updateGlobalParametersCmd = &cobra.Command{
+	Use:   "globalParameters",
```

Comment:
Since in the mta.yaml it's just called `parameters` and not `global-parameters` please change the command name to `parameters` instead of `globalParameters` (also the documentation, messages etc)